src/phoenix/server/api/input_types/ClearProjectInput.py

```diff
@@ -0,0 +1,12 @@
+from datetime import datetime
+from typing import Optional
+
+import strawberry
+from strawberry import UNSET
+from strawberry.relay import GlobalID
+
+
+@strawberry.input
+class ClearProjectInput:
+    id: GlobalID
+    end_time: Optional[datetime] = UNSET
```

Comment:
maybe add quick note to eliminate any ambiguity about which side of the cutoff is being deleted (even though one side makes more sense than the other)